manager-bundle/tests/HttpKernel/ContaoKernelTest.php

```diff
@@ -102,6 +102,9 @@ public function testRegisterBundles(): void
         $this->assertArrayNotHasKey(AppBundle::class, $bundles);
     }
 
+    /**
+     * @runInSeparateProcess
```

Comment:
Is this really necessary?

Reply:
Yes, because including the `AppBundle` file affects global state (IIRC).

Reply:
It is because of the `include`.